macros/audit.sql

```diff
@@ -93,9 +93,14 @@
         )}}
 {% endmacro %}
 
-{% macro unload_to_s3(unload_to_s3=False) %}
-  {% if unload_to_s3 %}
-    unload ('select dil.* from dbt_spectrum_redshift.dbt_invocation_logs dil union select d.* from {{ logging.get_audit_relation() }} d') to 's3://redshift-dbt-logs/dbt_invocation_logs/dbt_invocation_logs_' iam_role '{{ var('dbt_iam_role') }}' manifest delimiter as ',' null as '' escape allowoverwrite
+{% macro unload_to_s3(run_dbt_logging=False, unload_to_s3=False) %}
+
+  {% if run_dbt_logging %}
+    {% if unload_to_s3 %}
```

Comment:
so there's no case where you'd `unload` without logging?

Reply:
just out of curiosity, is it possible to do `{% if x and y %} in jinja?

Reply:
@huisben yeah, for now this is only capable of unloading this table after it logs. Not a huge limitation, but something we may find ourselves wanting to update in the future.

Reply:
@huisben yep! more complex [logic](http://jinja.pocoo.org/docs/2.10/templates/#logic) is available for `if` statements in Jinja. The reason I wrote it the way I did is to enable adding more specific functionality in the outer loop in the future if the need arises. But honestly it could be whatever we thing is best.